airflow/utils/log/file_task_handler.py

```diff
@@ -39,12 +38,20 @@ class FileTaskHandler(logging.Handler):
     :param base_log_folder: Base log folder to place logs.
     :param filename_template: template filename string
     """
-    def __init__(self, base_log_folder: str, filename_template: str):
+    def __init__(
+        self,
+        base_log_folder: str,
+        filename_template: str,
+        worker_server_log_port: int,
```

Comment:
The config option is called `worker_log_server_port` - did you give this a slightly different name on purpose?

Reply:
Fixed.